ext/XS-Typemap/t/Typemap.t

```diff
@@ -1,5 +1,5 @@
 BEGIN {
-    require Config; Config->import;
+    use Config;
```

Comment:
this is a correct usage of `use Config` as not `@INC` setup is performed, in this case no reasons to leave it in `BEGIN` block, no?